### PR TITLE
Send events

### PR DESCRIPTION
**Pull request by klzns**
Continuation of #4 and #5

### CODE REVIEW

**Comment by ingmoyete** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [x]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: docs/README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ This app provides a native integration for [Oct8ne](https://oct8ne.com/) custome
 
 1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) the `vtex.oct8ne@1.x` app in the desired account using your terminal. You can also install the app through the VTEX App Store.
 2. In the admin's account, access the **Apps** section and then select the **Oct8ne** box
-3. Fill out the form with your **Oct8ne license ID and baseURL** and save your changes.
+3. Fill out the form with your **Oct8ne license, server, and baseURL** and save your changes.
```
Comment (guerreirobeatriz) [inline docs/README.md:9]:
```suggestion
3. Fill out the form with your **Oct8ne license, server, and baseURL**.
4. Save your changes.
```

**Comment by ingmoyete** [comment]
> Fails
> 🚫	
> We follow [keepachangelog](https://keepachangelog.com/) standards.
> 
> Please, change the file `CHANGELOG.md` adding a small summary of your changes under the `[Unreleased]` section. Example:
> 
> ```diff
>  ## [Unreleased]
> 
> + ### Fixed
> + - Issue with image sizing in the SKU Selector.
> ```
> 
> Possible types of changes:
> 
> * `Added` for new features.
> * `Changed` for changes in existing functionality.
> * `Deprecated` for soon-to-be removed features.
> * `Removed` for now removed features.
> * `Fixed` for any bug fixes.
> * `Security` in case of vulnerabilities.
> 
> Warnings
> ⚠️	
> 👀 The size of this pull request seems relatively large (**>420** modifications). Consider splitting it into smaller pull requests to help make reviews easier and faster.
> 
> Generated by 🚫 [dangerJS](https://danger.systems/js) against [945ce4a](https://github.com/vtex-apps/oct8ne/commit/945ce4ae2d99e847150229bc07296d50c165fc21)

Hi, I cannot commit changes in the "Events" branch. Could you add the content below in the changelog for me?
-----------------------
### Added

- Customer data information available in the coviewer.
- Enable ecommerce proxy communication.

-------------------

**Comment by klzns** [comment]
One last thing @ingmoyete it's that a major version bump? 

Updating a major version will require stores that have this installed to update it manually by installing it again.

**Comment by ingmoyete** [comment]
> One last thing @ingmoyete it's that a major version bump?
> 
> Updating a major version will require stores that have this installed to update it manually by installing it again. If it breaks compatibility, then that's fine.
> 
> If there's no compatibility issues we can bump a minor version and the update will be automatically.

Yes, major version is ok. thanks for the info

**Comment by ingmoyete** [comment]
Hi,
would it be possible to merge this into master so i can test in my store?

**Comment by klzns** [comment]
@ingmoyete released!

**Pull request merged**